tensorflow_addons/utils/keras_utils.py

```diff
@@ -48,11 +48,16 @@ def normalize_tuple(value, n, name):
         return (value,) * n
     else:
         try:
+            try:
+                obj_value = iter(value)
+            except TypeError as te:
+                print (value, ' is not iterable')
+
```

Comment:
Why we need this?

Reply:
When filter_shape is None(not [None]), value_tuple = tuple(value) can cause the error `NoneType' object is not iterable. I think it looks not good. I would like to ask your suggestion of this issue.

Reply:
![Screenshot from 2019-10-04 17-38-33](https://user-images.githubusercontent.com/11615393/66247050-db185580-e6cd-11e9-808a-29aa8202bb2b.png)
But it still raises `TypeError`, and will be caught by the following `except: TypeError`.

Reply:
Thanks @WindQAQ.